x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/grouping/Bucket.java

```diff
@@ -92,6 +92,10 @@ public class Bucket extends GroupingFunction implements Validatable, TwoOptional
         examples = {
             @Example(
                 description = """
+                    `BUCKET` can work in two modes: one in which the size of the bucket is computed
+                    based on a buckets count recommendation (four parameters) and a range and
```

Comment:
nit: hard to grok the long sentence, maybe a comma would help?
```suggestion
                    based on a buckets count recommendation (four parameters) and a range, and
```

Reply:
Added.